src/core/mesh/qgsmeshtracerenderer.cpp

```diff
@@ -429,6 +430,9 @@ void QgsMeshStreamField::addTrace( QPoint startPixel )
   if ( !mVectorValueInterpolator )
     return;
 
+  if ( !( mMaximumMagnitude > 0 ) )
```

Comment:
mVectorDatasetMagMaximum <= 0

Reply:
I found this issue without ticket. Maybe fix #35888, but very not sure.